apps/extension/src/App/Accounts/DeleteAccount.tsx

```diff
@@ -94,7 +94,7 @@ export const DeleteAccount = (): JSX.Element => {
       >
         <Stack className="flex-1 justify-center" gap={4} full>
           <Stack as="header" gap={4}>
-            <Alert type="warning" title="Alert!">
+            <Alert type="warning" title="Alert!" className="max-w-84">
```

Comment:
@jurevans could you explain why this `max-w-84` was necessary in these updates?

Reply:
Yes! It was causing the layout to completely break for these specific components. I'm not sure why it doesn't affect Chrome, but the popup would get very wide without it. This value works for both Chrome & FF

Reply:
Oh gotcha! Thanks :)